src/client.ts

```diff
@@ -522,44 +596,64 @@ export class Tier {
       req.at = at
     }
     req.clobber = !!clobber
-
     return await this.apiPost<ReportRequest>('/v1/report', req)
   }
 
   public async subscribe(
     org: OrgName,
     features: Features | Features[],
-    effective?: Date,
-    info?: OrgInfo
+    { effective, info, trialDays, checkout }: SubscribeOptions = {}
   ): Promise<{}> {
-    // deprecated overloading of subscribe() as schedule()
-    if (
-      Array.isArray(features) &&
-      !features.some(f => !isPhase(f)) &&
-      effective === undefined &&
-      info === undefined
-    ) {
-      const msg = `Using phase objects with subscribe() is deprecated, please use tier.schedule() for this use case.`
-      process.emitWarning(msg, 'DeprecationWarning')
-      return await this.schedule(org, features as unknown as Phase[])
-    }
-
     const phases: Phase[] = !Array.isArray(features)
       ? [{ features: [features], effective }]
       : features.length
       ? [{ features, effective }]
       : []
-    return await this.schedule(org, phases, info)
+
+    if (trialDays && (typeof trialDays !== 'number' || trialDays <= 0)) {
+      throw new TypeError('trialDays must be number >0 if specified')
+    }
+    if (trialDays && phases.length) {
+      const real = phases[0]
+      /* c8 ignore start */
+      const effective = real.effective || new Date()
+      /* c8 ignore stop */
+      real.effective = new Date(
+        effective.getTime() + 1000 * 60 * 60 * 24 * trialDays
+      )
+      phases.unshift({ ...real, trial: true, effective })
+    }
+
+    return await this.schedule(org, phases, { info, checkout })
```

Comment:
Was this supposed to cross into the final line?

Reply:
I'm not sure what you mean?

Reply:
![CleanShot 2023-01-13 at 15 00 31@2x](https://user-images.githubusercontent.com/46/212434540-c4565c6d-5bf0-4291-8934-d02dacf3f7a8.png)

Reply:
Oh is that just GitHub wrapping the diff?

Reply:
Haha, yeah, looks like it.  It's all one line here.